velox/exec/tests/utils/AggregationFuzzer.cpp

```diff
@@ -1046,6 +1063,14 @@ velox::test::ResultOrError AggregationFuzzer::execute(
           .config(core::QueryConfig::kTestingSpillPct, "100");
     }
 
+    if (injectPartialSpill) {
+      spillDirectory = exec::test::TempDirectoryPath::create();
+      builder.spillDirectory(spillDirectory->path)
+          .config(core::QueryConfig::kSpillEnabled, "true")
+          .config(core::QueryConfig::kPartialAggregationSpillEnabled, "true")
+          .config(core::QueryConfig::kTestingSpillPct, "100");
+    }
```

Comment:
Flushing will be disabled when partial spill is on so this new dimension is needed to avoid losing coverage.